src/HPolyhedron.jl

```diff
@@ -499,15 +499,18 @@ The `HPolyhedron` (resp. `HPolytope`) obtained by the concrete convex hull of
 
 ### Notes
 
+It is suggested to use the `CDDLib.Library()` backend.
```

Comment:
Why is that?

Reply:
I saw that `CDDLib` is faster than the default backend for `convexhull` by a large margin.
Example:

```julia
using LazySets, Polyhedra

for n in [2, 4, 10]
    @time begin
        V1 = VPolytope([rand(n) for i in 1:100])
        V2 = VPolytope([rand(n) for i in 1:100])
        convex_hull(V1, V2) # uses Polyhedra's default library
    end
end

0.414625 seconds (4.13 M allocations: 230.331 MiB, 12.61% gc time)
# ?
# ?

using CDDLib

for n in [2, 4, 10]
    @time begin
        V1 = VPolytope([rand(n) for i in 1:100])
        V2 = VPolytope([rand(n) for i in 1:100])
        convex_hull(V1, V2, backend=CDDLib.Library())
    end
end

  0.086622 seconds (132.72 k allocations: 6.416 MiB)
  0.033711 seconds (15.75 k allocations: 856.891 KiB)
  0.511388 seconds (17.09 k allocations: 1.092 MiB, 1.62% gc time)
```

(times are shown on the second pass, and when there is a `?` is because i stopped the calculation)